backend/migrations/versions/57f694383a69_tags_db_roles_commented_out.py

```diff
@@ -0,0 +1,88 @@
+"""tags db, roles commented out
+
+Revision ID: 57f694383a69
+Revises: 09db37a849a4
+Create Date: 2025-01-28 10:32:14.365514
+
+"""
+from typing import Sequence, Union
+
+from alembic import op
+import sqlalchemy as sa
+import pgvector
+from sqlalchemy.dialects import postgresql
+
+# revision identifiers, used by Alembic.
+revision: str = '57f694383a69'
+down_revision: Union[str, None] = '09db37a849a4'
+branch_labels: Union[str, Sequence[str], None] = None
+depends_on: Union[str, Sequence[str], None] = None
+
+
+def upgrade() -> None:
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.create_table('permissions',
+    sa.Column('id', sa.UUID(), server_default=sa.text('gen_random_uuid()'), nullable=False),
+    sa.Column('area', sa.String(length=255), nullable=False),
+    sa.Column('type', sa.Enum('Self', 'Others', name='permissiontypeenum'), nullable=False),
+    sa.Column('access', sa.String(length=255), nullable=False),
+    sa.Column('description', sa.Text(), nullable=True),
+    sa.PrimaryKeyConstraint('id'),
+    schema='public'
+    )
+    op.create_table('tags',
+    sa.Column('id', sa.UUID(), server_default=sa.text('gen_random_uuid()'), nullable=False),
+    sa.Column('name', sa.String(length=255), nullable=False),
+    sa.Column('visibility', sa.Enum('public', 'private', name='visibilityenum'), nullable=False),
+    sa.Column('course_id', sa.UUID(), nullable=True),
+    sa.Column('parent_tag_id', sa.UUID(), nullable=True),
+    sa.Column('created_by', sa.UUID(), nullable=True),
+    sa.Column('properties', postgresql.JSONB(astext_type=sa.Text()), nullable=True),
+    sa.Column('created_at', sa.DateTime(timezone=True), server_default=sa.text('now()'), nullable=True),
+    sa.Column('updated_at', sa.DateTime(timezone=True), server_default=sa.text('now()'), nullable=True),
+    sa.ForeignKeyConstraint(['course_id'], ['public.courses.id'], ondelete='CASCADE'),
+    sa.ForeignKeyConstraint(['created_by'], ['auth.users.id'], ondelete='SET NULL'),
+    sa.ForeignKeyConstraint(['parent_tag_id'], ['public.tags.id'], ondelete='CASCADE'),
+    sa.PrimaryKeyConstraint('id'),
+    schema='public'
+    )
+    op.create_foreign_key(None, 'chunk_relations', 'chunks', ['target_chunk_id'], ['id'], source_schema='public', referent_schema='public', ondelete='CASCADE')
+    op.create_foreign_key(None, 'chunk_relations', 'chunks', ['source_chunk_id'], ['id'], source_schema='public', referent_schema='public', ondelete='CASCADE')
+    op.create_foreign_key(None, 'chunks', 'chunks', ['parent_chunk_id'], ['id'], source_schema='public', referent_schema='public', ondelete='CASCADE')
```

Comment:
Please verify the commands. alembic isn't perfect. im trying to make it not have this issue but at least cant blame them: "    # ### commands auto generated by Alembic - please adjust! ###"


most of the create_foreign_key and the corresponding  drop are unrelated to you your changes. please delete those lines